www/includes/easyparliament/templates/html/alert/_mp_alert_form.php

```diff
@@ -0,0 +1,114 @@
+          if ($row['constituency']) {
+              $name .= ' (' . gettext($row['constituency']) . ')';
+          }
+          printf(gettext('When %s speaks.'), $name);
+          ?>
+              <input type="submit" class="button small" value="<?= gettext('Subscribe') ?>"></form>
+          </form>
+      </li>
+    <?php } ?>
+  </ul>
+  <hr />
+<?php } else { ?>
+  <form action="<?= $actionurl ?>" method="post" class="alerts-form">
+    <?php if (!$email_verified) { ?>
+      <p>
+        <?php if (isset($errors["email"]) && $submitted) { ?>
+          <span class="alert-page-error"><?= $errors["email"] ?></span>
+        <?php } ?>
+          <input type="email" class="form-control" placeholder="<?= gettext('Your email address') ?>" name="email" id="email" value="<?= _htmlentities($email) ?>">
+      </p>
+    <?php } ?>
+
+    <?php if ($token) { ?>
+        <input type="hidden" name="t" value="<?= _htmlspecialchars($token) ?>">
+    <?php } ?>
+
+      <p>
+        <?php if ($pid) { ?>
+          <input type="text" class="form-control" name="mp_search" id="mp_search" disabled="disabled"
+              value="<?= $pid_member->full_name() ?><?php if ($pid_member->constituency()) { ?> (<?= _htmlspecialchars($pid_member->constituency()) ?>)<?php } ?>">
+        <?php } elseif ($keyword) { ?>
+          <input type="text" class="form-control" name="mp_search" id="mp_search" value="<?= _htmlspecialchars($keyword) ?>">
+        <?php } else { ?>
+          <label for="mp-postcode"><?= gettext('Search by postcode, or MP/MSP/MS/MLA name') ?></label>
+          <input id="mp-postcode" type="text" class="form-control" placeholder="<?= gettext('e.g. B2 4QA or John Doe') ?>" name="mp_search" id="mp_search" value="<?= _htmlentities($search_text) ?>" style="min-width:300px;">
+        <?php } ?>
+      </p>
+
+      <div class="alert-form__section">
+        <div class="checkbox-wrapper">
+          <input type="checkbox" id="ignore_speaker_votes" name="ignore_speaker_votes"<?= $ignore_speaker_votes ? ' checked' : ''?>>
+          <label for="ignore_speaker_votes"><?= gettext('Do not include votes in the alert') ?></label>
+        </div>
+      </div>
+
+      <p>
+          <?php if ($pid || $keyword) { ?>
+          <button type="submit" class="button" name="mp_step" value="mp_confirm">
+            <i aria-hidden="true" class="fi-megaphone"></i>
+            <span><?= gettext('Subscribe') ?></span>
+          </button>
+          <?php } else { ?>
+          <button type="submit" class="button" name="mp_step" value="mp_search">
+            <span><?= gettext('Search') ?></span>
+            <i aria-hidden="true" class="fi-magnifying-glass"></i>
+          </button>
+          <?php } ?>
+          <button type="submit" class="button button--red" name="action" value="Abandon">
+            <i aria-hidden="true" class="fi-trash"></i>
+            <span><?= gettext('Abandon changes') ?></span>
```

Comment:
It's still saying "Abandon changes" if I go to /alert/ and click the rep alert button...